interfaces/cython/cantera/cti2yaml.py

```diff
@@ -1632,9 +1631,19 @@ def convert(filename=None, output_name=None, text=None):
         else:
             # Error in cti2yaml or elsewhere
             traceback.print_exc()
-
         sys.exit(4)
 
+    # get file description from header block
+    description = []
+    for line in text.splitlines():
+        # only consider comments in the initial contiguous comment block
+        # comments start with '#'; there may be empty leading lines
+        if line.startswith("#") or not description:
+            description.append(line[1:].rstrip())
+        else:
+            break
```

Comment:
What if you have multiple leading blank lines? On line 0, you'd get `description` is empty, so you'd append an empty string to `description`. On line 1, you'd find it doesn't start with `#` and `description` isn't empty, so this would `break`.

Reply:
Fair enough. Fixed and covered by test.